SampleApp/src/main/java/ly/kite/sample/MainActivity.java

```diff
@@ -0,0 +1,176 @@
+package ly.kite.sample;
+
+import com.crashlytics.android.Crashlytics;
+import io.fabric.sdk.android.Fabric;
+import java.net.URL;
+import java.util.ArrayList;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Switch;
+import android.widget.Toast;
+
+import ly.kite.print.Asset;
+import ly.kite.KiteSDK;
+
+public class MainActivity extends Activity
+  {
+
+  /**********************************************************************
+   * Insert your Kite API keys here. These are found under your profile
+   * by logging in to the developer portal at https://www.kite.ly
+   **********************************************************************/
+  //private static final String API_KEY_TEST = "ba171b0d91b1418fbd04f7b12af1e37e42d2cb1e";
+  private static final String API_KEY_TEST = "0453d74be957c1eb510fc2d580007294cdc31a79"; // Photobox
```

Comment:
Could you move to API_KEY_TEST = "REPLACE_ME" -- would prefer to avoid leaking any internal or partner keys. Also the // Photobox comment